src/server/db/db_helper.py

```diff
@@ -438,9 +438,15 @@ async def validate_sql(self, db_name, query):
             # return False
 
 class sqlite_database(Database):
+    def __init__(self):
+        self.temp_db_folder = "temporary_db"
+        if not os.path.exists(self.temp_db_folder):
+            os.makedirs(self.temp_db_folder)
```

Comment:
Dont use temp_db_folder as variable name. Give it a more descriptive name

Reply:
done